host/xtest/pkcs11_1000.c

```diff
@@ -1930,6 +2035,20 @@ static void xtest_pkcs11_test_1009(ADBG_Case_t *c)
 				goto err_destr_obj;
 		}
 
+		if (test->in != NULL) {
```

Comment:
I suggest you use the instruction block above.
```patch
 			if (!ADBG_EXPECT_CK_RESULT(c, CKR_SIGNATURE_LEN_RANGE,
 						   rv))
 				goto err_destr_obj;
-		}
-
-		if (test->in != NULL) {
 			rv = C_VerifyInit(session, test->mechanism, key_handle);
 		(...)

Reply:
ok